models/cart-discount/src/cart-discount-draft/presets/sample-data-fashion/percent-off-womens-wear.ts

```diff
@@ -0,0 +1,33 @@
+import * as CartDiscountValueRelative from '@commercetools-test-data/cart-discount-value-relative';
+import { LocalizedString } from '@commercetools-test-data/commons';
+import * as CartDiscountDraft from '../..';
+import { TCartDiscountDraftBuilder } from '../../../types';
+
+const percentOffWomensWear = (): TCartDiscountDraftBuilder =>
+  CartDiscountDraft.presets
+    .empty()
+    .value(
+      CartDiscountValueRelative.CartDiscountValueRelativeDraft.random()
+        .type('relative')
```

Comment:
Random thought, given the entity is `CartDiscountValueRelativeDraft` should `type('relative')` be a given? 

Care amount so low, throwing things out into the universe.

Reply:
You're not wrong 😅  017343388571b114ad5d5b557254cb091cf9bf13